#> cat localmap.msg

float32 xpos
float32 ypos
obstacle_info[] obstacle_data